doc-content/enterprise-only/bpmn/bpmn-gateways-ref.adoc

```diff
@@ -1,104 +1,77 @@
+image:BPMN2/gateway.png[]
+
+In this example, the customer details are verified by a user and the process is assigned to a user for approval. If approved, an approval notification is sent to the user. If the event of the request is rejected, a rejection notification is sent to the user.
 
 
 .Gateway elements table
-[cols="4"]
+[cols="2"]
 |===
-.2+h|
-.2+h|Top-level
-2+h|Subprocesses
 
 
-h|Interrupt
-h|Non-interrupt 
+|exclusive (XOR)
+|image:BPMN2/bpmn-gateway-exclusive.png[]
 
-|Exclusive (XOR)
-|image:BPMN2/bpmn-start-node.png[]
-|
-|
 
 |Inclusive
-|
-|
-|
+|image:BPMN2/bpmn-gateway-inclusive.png[]
+
+
 
 |Parallel
-|
-|
-|
+|image:BPMN2/bpmn-gateway-parallel.png[]
+
 
 |Event
-|
-|
-|
+|image:BPMN2/grn_star_lg.png[]
+
 
 |===
 
 [[_exclusive_gateway]]
 .Exclusive
-In an exclusive converging gateway, only the first incoming flow whose condition evaluates to true is chosen. In a diverging gateway only one outgoing Flow is chosen.
+In an exclusive diverging gateway, only the first incoming flow whose condition evaluates to true is chosen. In a converging gateway, the next node is triggered for each triggered incoming flow.
 
 The gateway triggers exactly one outgoing flow. The flow with the constraint evaluated to true and the _lowest_ priority number is taken.
 
 
 [IMPORTANT]
 ====
-Ensure that at least one of the outgoing flow evaluates to true at runtime. Otherwise, the process instance terminates with a runtime exception.
+Ensure that at least one of the outgoing flows evaluates to true at runtime. Otherwise, the process instance terminates with a runtime exception.
 ====
 
 
-The converging gateway allows a workflow branch to continue to its outgoing flow as soon as it reaches the gateway. When one of the incoming flows triggers the gateway, the workflow continues to the outgoing flow of the gateway. If it is triggered from more than one incoming flow, it triggers the next node for each trigger.
-
-
-* Attributes
-*Default gate*
-+
-The outgoing flow taken by default if no other flow can be taken.
+The converging gateway enables a workflow branch to continue to its outgoing flow as soon as it reaches the gateway. When one of the incoming flows triggers the gateway, the workflow continues to the outgoing flow of the gateway. If it is triggered from more than one incoming flow, it triggers the next node for each trigger.
 
 
 [[_inclusive_gateway]]
 .Inclusive
 
-With a diverging inclusive gateway, the incoming flow is taken, all outgoing flows that evaluate to true are taken.
-Connections with lower priority numbers are triggered before triggering higher priority ones. Priorities are evaluated but the BPMN2 specification does not guarantee the priority order. It is recommended that you do not depend on the `priority` attribute in your workflow.
+With a diverging inclusive gateway, the incoming flow is taken and all outgoing flows that evaluate to true are taken.
```

Comment:
"With an inclusive diverging gateway, ..." Since above it was "exclusive diverging gateway."? Search all for "diverging inclusive" and switch and confirm consistency with "exclusive diverging" etc. Plus makes more sense, to me. Unless it's a technical term as is, then maybe the above "exclusive" one is off. 

Same with "diverging parallel gateway" vs "parallel diverging gateway" Need to decided which is meant and consistently use across all.